pkgs/by-name/wa/walksnail-osd-tool/make-build-reproducible.patch

```diff
@@ -0,0 +1,35 @@
+--- a/ui/build.rs	2024-04-26 16:11:59.502431471 +0200
++++ b/ui/build.rs	2024-04-26 16:12:44.539532395 +0200
+@@ -5,28 +5,11 @@
+ fn main() {
+     println!("cargo:rerun-if-changed=./src");
+
+-    // Save details from build environment so they can be included in the binary
+-    vergen(Config::default()).unwrap();
++    println!("cargo:rustc-env=VERGEN_CARGO_TARGET_TRIPLE=x86_64-unknown-linux-gnu");
++    println!("cargo:rustc-env=VERGEN_RUSTC_SEMVER=nixos-unstable-latest");
++    println!("cargo:rustc-env=GIT_VERSION=v0.3.0");
++    println!("cargo:rustc-env=GIT_COMMIT_HASH=1044026bab773386fe96e20b544f60ad73d24e3f");
```

Comment:
Put these variables into `env`, like `OPENSSL_NO_VENDOR`. Hardcoding version and hashes would break automated updating scripts. `VERGEN_CARGO_TARGET_TRIPLE` should not be tied to a particular architecture (x86_64).

Also, does it need each of these variables at all?